test/configCases/web/prefetch-preload-module/index.js

```diff
@@ -0,0 +1,90 @@
+// This config need to be set on initial evaluation to be effective
```

Comment:
Just copy/paste test for non ESM prefetch/preload and change `rel` on valid value, also there is good test for ESM modules (we don't have many them :smile: )